jax/_src/numpy/lax_numpy.py

```diff
@@ -1865,8 +1866,12 @@ def _reduction_init_val(a, init_val):
 
 _cast_to_bool = partial(lax.convert_element_type, new_dtype=bool_)
 
-sum = _make_reduction("sum", np.sum, lax.add, 0, upcast_f16_for_computation=True,
-                      bool_op=lax.bitwise_or)
+def sum(a, axis=None, dtype=None, out=None, keepdims=False, use_kahan_summation=False):
```

Comment:
I might err on the side of *not* adding this feature to `jax.numpy.sum` for now. We want to be somewhat cautious about extensions to the numpy api.

Reply:
I removed it from numpy, but exposed a `lax.reduce_sum` with the `use_kahan_summation` kwarg, if that's a better alternative.